components/scream/src/physics/cld_fraction/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(CLDFRAC_SRCS
+  atmosphere_cld_fraction.cpp
+  cld_fraction.cpp 
+)
+
+set(CLDFRAC_HEADERS
+  atmosphere_cld_fraction.hpp
+  cld_fraction_functions.hpp
+  cld_fraction_main_impl.hpp
+)
+
+add_library(cld_fraction ${CLDFRAC_SRCS})
+target_include_directories(cld_fraction PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../share)
+target_link_libraries(cld_fraction physics_share scream_share)
+target_compile_options(cld_fraction PUBLIC)
+
+#if (NOT SCREAM_LIB_ONLY)
```

Comment:
So I guess there's no unit test of cld_fraction? In particular, no bfb testing of v1-vs-v0? Is that just something we plan to do later, or are we ok without it?

Reply:
@PeterCaldwell can comment.  I made the decision to let the cld_fraction_stand_alone to take care of these sorts of tests.  As for BFB, the change I made to F90 was more surgical.  I think we may still be able to make a BFB test, but it would be a fair bit of work.  Not sure if it is high priority.

Reply:
I'm conflicted about whether we need a BFB test for cld_fraction. On one hand, BFB tests are always good. On the other hand, our impl is only 3 lines of code or so, so it seems unlikely we'd screw it up. And the F90 version is entwined in a much bigger ice cloud fraction scheme that we don't want to port, while the F90 total cloud fraction calculation is stuffed into shoc_intr.F90. This makes creating a BFB test annoying. Thus I really don't think it should be part of this PR, but we might want to create a follow-on PR to do the BFB testing.

Reply:
At the risk of adding one more thing to the "TODO" pile, we could certainly open up a git-issue to add in a BFB test.  I'm in favor of not creating one now just because there are a number of higher priority items to address in pursuit of v1 and I'm pretty confident (given that its only a few lines) that the implementation of ice cloud fraction is correct.  As Peter pointed out, a BFB test with the F90 would probably take a few days and delay this PR going in.

Reply:
Sounds good to me. If the F90 impl is buried deep into other F90 routines, then it's not a trivial task, and one needs to weigh the benefits and costs. Aaron, you could remove those lines from the CMakeLists.txt, and add a 2 lines comment explaining why this folder does not have a 'tests' subdir, explaining what we discussed here.

Reply:
done